addons/point_of_sale/models/pos_order.py

```diff
@@ -304,20 +304,18 @@ def _get_pos_anglo_saxon_price_unit(self, product, partner_id, quantity):
     is_invoiced = fields.Boolean('Is Invoiced', compute='_compute_is_invoiced')
     is_tipped = fields.Boolean('Is this already tipped?', readonly=True)
     tip_amount = fields.Float(string='Tip Amount', digits=0, readonly=True)
-    refund_orders_count = fields.Integer('Number of Refund Orders', compute='_compute_refund_related_fields')
-    is_refunded = fields.Boolean(compute='_compute_refund_related_fields')
-    refunded_order_ids = fields.Many2many('pos.order', compute='_compute_refund_related_fields')
+    refund_orders_count = fields.Integer('Number of Refund Orders', compute='_compute_refund_related_fields', help="Number of orders where items from this order were refunded")
+    refunded_order_id = fields.Many2one('pos.order', compute='_compute_refund_related_fields', help="Order from which items were refunded in this order")
```

Comment:
I had to read the help 2-3 times even knowing what the field does. 
This is correct but not clear, i think we should change it.

Something like: "Order linked to the refunded items".
Maybe ask Dama

Reply:
I agree, this is unclear. I'll change it to your suggestion. Do you agree, @dama-odoo ?